File: Back/AMN/src/main/java/com/ibkpoc/amn/entity/Meeting.java
package com.ibkpoc.amn.entity;

import jakarta.persistence.*;
import lombok.*;

import java.time.LocalDateTime;

@Entity
@Table(name = "ibk_poc_conf")
@Data
@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
public class Meeting {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long confId;

    private String title;
    private LocalDateTime startTime;
    private LocalDateTime endTime;
    private Boolean isActive;
    private String summary;
    @Column(name = "overall_summary")
    private String overallSummary; // 전체 요약 추가

    private String topic;
    @Column(name = "participant_count") // 컬럼명 명시
    private Integer participants; // 참가자 수
    private String wavSrc; // 녹음 파일 위치 추가
    private String sttSrc; // 녹음 파일 위치 추가
    private Boolean sttSign;
}

File: 앱/IBK_PoC/app/src/main/java/com/ibkpoc/amn/model/Meeting.kt
// model/Meeting.kt 수정 (기존 모델 유지하면서 확장)
package com.ibkpoc.amn.model

data class Meeting(
    val id: Long,
    val title: String,           // "회의 ${logId}"와 같은 형식으로 생성
    val start: String,
    val end: String,
    val participantCount: Int,
    val topic: String
)